.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+node_js:
+ - "stable"
+install: npm install
+
+services:
+- postgresql
+
+cache:
+  directories:
+    - "node_modules"
+
+before_script:
+  - psql -c 'create database travis_ci_test;' -U postgres
+
+script: npm run test
+
+after_success:
+  - npm run coverage
```

Comment:
New line required at the end of the file

Reply:
Not treated accordingly yet, there is an extra space currently included

Reply:
treated accordinly